### PR TITLE
chore(Farms): UI improvement

### PR DESCRIPTION
**Pull request by Woody810**
Use ApyLabelContainer for the APR of the LP position

### CODE REVIEW

**Comment by chef-jojo** [comment]
It's not external

File: apps/web/src/views/Farms/components/FarmCard/V3/FarmV3StakeAndUnStake.tsx [hunk 1]
```diff
@@ -49,9 +48,10 @@ export const FarmV3LPTitle = ({
   title: string
   outOfRange: boolean
 }) => (
-  <StyledLink external href={liquidityUrl}>
```
Comment (Woody810) [inline apps/web/src/views/Farms/components/FarmCard/V3/FarmV3StakeAndUnStake.tsx:52]: Should we remove the external here?

Comment (chefilip) [reply apps/web/src/views/Farms/components/FarmCard/V3/FarmV3StakeAndUnStake.tsx:52]: I think we should keep it though. I prefer keeping user at where they are.

Comment (Woody810) [reply apps/web/src/views/Farms/components/FarmCard/V3/FarmV3StakeAndUnStake.tsx]: Part of the LP position has been restored.

Comment (chefilip) [reply apps/web/src/views/Farms/components/FarmCard/V3/FarmV3StakeAndUnStake.tsx]: I see. It's good to remove it then

**Review by chefilip — APPROVED** [review]

File: apps/web/src/views/Farms/components/FarmCard/V3/FarmV3StakeAndUnStake.tsx (hunk 1 above)
Comment (Woody810) [reply apps/web/src/views/Farms/components/FarmCard/V3/FarmV3StakeAndUnStake.tsx]: merged..lol

just remove it on https://github.com/pancakeswap/pancake-frontend/pull/6523

**Pull request merged**